release-notes/10.0/preview/preview1/winforms.md

```diff
@@ -1,14 +1,70 @@
+
+**Clipboard:**
+
+- `public object GetData(string format)`
+
+**DataObject:**
+
+- `public virtual object? GetData(string format)`
+- `public virtual object? GetData(string format, bool autoConvert)`
+- `public virtual object? GetData(Type format)`
+
+**ClipboardProxy (VisualBasic wrapper for WinForms Clipboard):**
+
+- `public object GetData(string format)`
+
+### New APIs
+
+We’ve introduced the following APIs:
+
+**Clipboard:**
 
-- [What's new in Windows Forms in .NET 10](https://learn.microsoft.com/dotnet/desktop/winforms/whats-new/net100) documentation
+- `public void SetDataAsJson<T>(T data) { }`
+- `public void SetDataAsJson<T>(string format, T data) { }`
+- `public void SetDataAsJson<T>(string format, bool autoConvert, T data) { }`
+- `public virtual bool TryGetData<T>(out T data) { }`
+- `public virtual bool TryGetData<T>(string format, out T data) { }`
+- `public virtual bool TryGetData<T>(string format, bool autoConvert, out T data) { }`
+- `public virtual bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data) { }`
 
+**IDataObject:**
+
+- `bool TryGetData<T>(out T data);`
+- `bool TryGetData<T>(string format, out T data);`
+- `bool TryGetData<T>(string format, bool autoConvert, out T data);`
+- `bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data);`
+
+**Control:**
+
+- `public DragDropEffects DoDragDropAsJson<T>(T data, DragDropEffects allowedEffects, Bitmap? dragImage, Point cursorOffset, bool useDefaultDragImage)`
+- `public DragDropEffects DoDragDropAsJson<T>(T data, DragDropEffects allowedEffects)`
+
+**DragDrop:**
+
+- `public static DragDropEffects DoDragDropAsJson<T>(DependencyObject dragSource, T data, DragDropEffects allowedEffects)`
+
+**ClipboardProxy:**
+
+- `public void SetDataAsJson<T>(T data) { }`
+- `public void SetDataAsJson<T>(string format, T data) { }`
+- `public bool TryGetData<T>(string format, out T data) { }`
+- `public bool TryGetData<T>(string format, System.Func<System.Reflection.Metadata.TypeName, System.Type> resolver, out T data) { }`
+
+### New Configuration Switch
+
+If an application is unable to adopt the new APIs, and requires a fallback to BinaryFormatter serialization during Clipboard and Drag/Drop operations, the developer will need to take additional action. In .NET 9, calls to BinaryFormatter will always throw a `PlatformNotSupportedException` unless the developer follows the instructions *An external link was removed to protect your privacy.*. In .NET 10 Preview 1, WinForms will now also by default disable the use of BinaryFormatter as a fallback unless it is explicitly enabled with this new configuration switch. This switch is `ClipboardDragDrop.EnableUnsafeBinaryFormatterSerialization`, and it must be set to `True` for the fallback to happen successfully.

```

Comment:
Ditto here with external link. I think we may want to include this one: https://learn.microsoft.com/dotnet/standard/serialization/binaryformatter-migration-guide/compatibility-package